docs/docs/config.mdx

```diff
@@ -68,6 +68,7 @@ wsh editconfig
 | window:magnifiedblockblursecondarypx | int      | change the blur in CSS pixels that is applied to the visible portions of non-magnified blocks when a block is magnified (see [backdrop-filter](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) for more info on how this gets applied)      |
 | window:maxtabcachesize               | int      | number of tabs to cache. when tabs are cached, switching between them is very fast. (defaults to 10)                                                                                                                                                          |
 | window:showmenubar                   | bool     | set to use the OS-native menu bar (Windows and Linux only, requires app restart)                                                                                                                                                                              |
+| window:autohidetabsbar               | bool     | auto hide tab bar based on true or false (requires restart)
```

Comment:
This shouldn't require restart since it's fully on the frontend 

```suggestion
| window:autohidetabbar               | bool     | show and hide the tab bar automatically when the mouse moves near the top of the window
```